internal/characters/yanfei/asc.go

```diff
@@ -56,7 +56,7 @@ func (c *char) a4() {
 			HitlagFactor:       0.05,
 			CanBeDefenseHalted: defhalt,
 		}
-		c.Core.QueueAttack(ai, combat.NewCircleHit(c.Core.Combat.Player(), 0.1), 10, 10)
+		c.Core.QueueAttack(ai, combat.NewCircleHit(c.Core.Combat.Player(), 3.5), 10, 10)
```

Comment:
you should pull the target out of the OnEnemyDamage event and center the attack around that 
```
trg := args[0].(combat.Target)
...
c.Core.QueueAttack(ai, combat.NewCircleHit(trg, 3.5), 10, 10)
```